### DISCUSSION TITLE
Is it possible to declare the file location in the spreadsheet?

### DISCUSSION THREAD

**Discussion by michaellammers** [fross123/csv_to_qlab#35] [Q&A] [answered]
If my 'type' is 'audio' how can I set the file location on my hard drive? Is that possible using this method?

**Comment by fross123** [comment, marked as answer]
Thanks for the comment!

This is possible to implement, however it does not seem like it would save much time since there are a lot of parameters with audio cues that need to be changed for each cue regardless. Also, dragging the file in seems easier than keeping the directory path consistent in a spreadsheet, but that could just be me.

What is the use case for this feature? Maybe we can work backwards from something specific.

**Comment by michaellammers** [comment]
Thanks for getting back to me on this Finlay. The situation is as follows:

- I have 3 lists of 150 sentences (450 in total) spoken by a voice actor.
- Each list is a language with the narrations in their own folder, so 150 files in folder A, 150 files in folder B and 150 files in folder C.
- The narrations all have the same parameters except for name and number
- The name and number has to be set to a fixed value

One could drag them into qlab, but then have to go by each and every file to update the number and name manually, whereas with your solution could spare users in the same situation a lot of time.

Hope you can help!

**Reply by fross123** [reply]
Ok, I think I understand. I will look into adding a "file target" header. I do think the best way would be to include the full file path in the spreadsheet. I'll update here once I look into this a bit more.

Thanks!

**Reply by michaellammers** [reply]
That would be great! Full path seems to be the way to go, yes. I think this would come in quite handy and a real time saver in some situations. Highly appreciated 🙏

**Comment by michaellammers** [comment]
I was wondering if this is still on the roadmap or if we should look at another solution.

**Reply by fross123** [reply]
@michaellammers thank you for the nudge on this. Updated to include this feature with the latest version v2022.2. You should be able to add a "file target" header to your csv file, everything else should be the same. Please note that you will need to re-install csv-to-qlab.

Thanks!

**Reply by michaellammers** [reply]
Thank you so much! This is a huge time saver.
